.eslintrc.json

```diff
@@ -1,81 +0,0 @@
-{
```

Comment:
So this was converted with `biome migrate eslint --write` I assume.

https://biomejs.dev/guides/migrate-eslint-prettier/

Reply:
Yes, it was, however this also had the `--include-inspired` flag